### ISSUE TITLE
Understanding _joint_flip_mask for Aloha robot

### ISSUE THREAD

**Issue by swagh1611** [Physical-Intelligence/openpi#278]
First off, thank you for making this project open source! Your work has been incredibly valuable.

I'm currently working on configuring XArm 6, following the Aloha example, and fine-tuning it on my own data. While setting up the adaptation, I came across the **_joint_flip_mask** function, which is used when **adapt_to_pi** is enabled:
https://github.com/Physical-Intelligence/openpi/blob/main/src/openpi/policies/aloha_policy.py#L110

I’d like to understand the significance of this mask. Any guidance on this would be greatly appreciated! 
Thanks again for your contributions.

**Comment by uzhilinsky** [comment]
The short answer is that you don't have to worry about it, assuming that you are using the standard Aloha runtime.

The reason why the `adapt_to_pi` functionality is there in the first place, is because our internal runtime, which we use to collect data from Trossens, has a different convention where some of the joints are flipped (note that we are also transforming the gripper space). This is how the base model was trained. The code that gets enabled when `adapt_to_pi` is to True (which is the default), is taking care of transforming the standard Aloha data into that space and back.

**Comment by uzhilinsky** [comment]
I'll resolve since this is not an issue but please feel free to move this into "Discussions" if you have further questions.

**Comment by dingjiansw101** [comment]
> The short answer is that you don't have to worry about it, assuming that you are using the standard Aloha runtime.
> 
> The reason why the `adapt_to_pi` functionality is there in the first place, is because our internal runtime, which we use to collect data from Trossens, has a different convention where some of the joints are flipped (note that we are also transforming the gripper space). This is how the base model was trained. The code that gets enabled when `adapt_to_pi` is to True (which is the default), is taking care of transforming the standard Aloha data into that space and back.

@uzhilinsky I used a mobile aloha from Trossen. I could directly set the adapt_to_pi as False, right?

**Comment by wkMvg** [comment]
> The short answer is that you don't have to worry about it, assuming that you are using the standard Aloha runtime.
> 
> The reason why the `adapt_to_pi` functionality is there in the first place, is because our internal runtime, which we use to collect data from Trossens, has a different convention where some of the joints are flipped (note that we are also transforming the gripper space). This is how the base model was trained. The code that gets enabled when `adapt_to_pi` is to True (which is the default), is taking care of transforming the standard Aloha data into that space and back.

I want to know if i use a arm not in pretrained dataset how do i transform my joint to pi?
